app/models/bot/smooch.rb

```diff
@@ -884,9 +892,17 @@ def self.save_media_message(message, type = 'image')
 
       data = f.read
       hash = Digest::MD5.hexdigest(data)
-      filename = type == 'image' ? "#{hash}.jpeg" : "#{hash}.mp4"
+      filename =
+        if type == 'image'
+          "#{hash}.jpeg"
+        elsif type == 'video'
+          "#{hash}.mp4"
+        elsif type == 'audio'
+          "#{hash}.mp3"
+        end
```

Comment:
This can be simplified by just getting the extension based on the type... the name itself is the same for all.